app/controllers/admin/schedules_controller.rb

```diff
@@ -7,10 +7,11 @@ class SchedulesController < Admin::BaseController
     load_resource :venue, through: :conference, singleton: true
 
     skip_before_filter :verify_authenticity_token, only: [:update]
-    layout 'schedule'
 
     def show
-      authorize! :update, @program.events.new
+      event = @program.events.new
+      authorize! :update, event
+      event.destroy
```

Comment:
Yes, I instantiate the event to keep the authorization as it was, but then I destroy it because otherwise when I do `@program.save!` in line 29 I save the new event. Well, the validations fail, so I do not save it but it doesn't work.